expression/builtin_time_vec.go

```diff
@@ -172,12 +174,111 @@ func (b *builtinStringStringTimeDiffSig) vecEvalDuration(input *chunk.Chunk, res
 	return errors.Errorf("not implemented")
 }
 
+// Monday is 0, ... Sunday = 6 in MySQL
+// but in go, Sunday is 0, ... Saturday is 6
+// we will do a conversion.
+func weekdayConversion(weekday gotime.Weekday) int64 {
```

Comment:
Can we implement a ```vecEvalIndex``` function like [this](https://github.com/pingcap/tidb/blob/master/expression/builtin_time.go#L1087)?

Reply:
ok, I will do it